server/src/main/java/org/elasticsearch/search/SearchModule.java

```diff
@@ -343,9 +343,9 @@ private void registerAggregations(List<SearchPlugin> plugins) {
         registerAggregation(new AggregationSpec(SumAggregationBuilder.NAME, SumAggregationBuilder::new, SumAggregationBuilder::parse)
                 .addResultReader(InternalSum::new));
         registerAggregation(new AggregationSpec(MinAggregationBuilder.NAME, MinAggregationBuilder::new, MinAggregationBuilder::parse)
-                .addResultReader(InternalMin::new));
+                .addResultReader(InternalMin::new).setAggregatorRegistrar(MinAggregationBuilder::registerAggregators));
```

Comment:
Should we put these on the next line?  We should probably come up with an overall formatting for these anyhow, I think they are all over the place depending on the agg and the PR :)

Reply:
Aren't we supposed to have an autoformater for this now? ;)

I think I like putting it on a new line slightly better, not sure why I didn't do that here.  I'll push an update.